src/Behavioral.Automation/Bindings/TableBinding.cs

```diff
@@ -23,12 +24,23 @@ public void CheckRowsCount([NotNull] ITableWrapper element, int count)
         [Then("(.*?) should (contain|not contain) the following rows:")]
         public void CheckTableContainsRows(ITableWrapper gridRows, string behavior, Table table)
         {
-            var tableList = ListServices.TableToCellsList(table);
+            var expectedValues = ListServices.TableToCellsList(table);
+
             Assert.ShouldBecome(() => gridRows.Stale, false, $"{gridRows.Caption} is stale");
-            Assert.ShouldBecome(() => gridRows.Text.Contains(tableList.FirstOrDefault()), true, $"{gridRows.Caption} text is \"{gridRows.Text}\"");
```

Comment:
sometimes we have tables without caption, so this check can't be done inside this binding.